src/actions/go-to.ts

```diff
@@ -0,0 +1,22 @@
+import 'colors';
+import { Browser } from 'puppeteer';
+import Action from '../action.i';
+
+/**
+ * Opens a new window for a given browser and navigates to a specified url.
+ */
+
+export default class GoTo implements Action {
+  constructor() {}
+
+  // @ts-ignore
+  async execute(title: string, args: string[], outs: any[], context: any): Promise<any> {
+    var browser: Browser = context.browser;
+    let page = await browser.newPage();
+    await page.goto(args[0]).catch(err => console.log(title.red, err));
```

Comment:
Let's throw errors so that the framework can catch them.

Reply:
Done